README.md

```diff
@@ -1,84 +1,41 @@
-    - [ ] Check *Include Administrators*
-  - [ ] On the *Manage Access* tab add the appropriate groups
-- About Section (accessed on the main page of the repo, click the gear icon to edit)
-  - [ ] The repo should have a short description of what it is for
-  - [ ] Add one of the following topic tags:
-    | Topic Tag       | Usage                                    |
-    | --------------- | ---------------------------------------- |
-    | az              | For actions related to Azure             |
-    | code            | For actions related to building code     |
-    | certs           | For actions related to certificates      |
-    | db              | For actions related to databases         |
-    | git             | For actions related to Git               |
-    | iis             | For actions related to IIS               |
-    | microsoft-teams | For actions related to Microsoft Teams   |
-    | svc             | For actions related to Windows Services  |
-    | jira            | For actions related to Jira              |
-    | meta            | For actions related to running workflows |
-    | pagerduty       | For actions related to PagerDuty         |
-    | test            | For actions related to testing           |
-    | tf              | For actions related to Terraform         |
-  - [ ] Add any additional topics for an action if they apply    
-  - [ ] The Packages and Environments boxes can be unchecked
-
-  
+# delete-branch-package-versions
+
+This action will retrieve the list of versions for a GH Package in the provided organization, then delete any package versions with a name that matches the pre-release format generated by [im-open/git-version-lite] (`major.minor.patch-<sanitized-branch-name>.<formated-date>`).  
+  - To sanitize the branch name, the characters `a-z, A-Z, 0-9 and -` are kept and any other character is replaced with `-`.  
+
+If the action runs into an issue deleting a specific package version, it will generate a warning that can be viewed in the Summary section of the workflow rather than failing.  Errors retrieving the package versions will still cause the action to fail though.
+
 ## Inputs
-| Parameter | Is Required | Default | Description           |
-| --------- | ----------- | ------- | --------------------- |
-| `input`   | true        |         | Description goes here |
+| Parameter      | Is Required | Description                                                                                                                                                        |
+| -------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `github-token` | true        | A token with permission with the `packages:delete` and `packages:read` scopes that is authorized in the organization where the package versions to be deleted are. |
```

Comment:
Something about how this reads is throwing me off a bit, but maybe it's nothing. Here's a suggestion of what feels a bit better to me, but I'm not married to the idea of changing this.
```suggestion
| `github-token` | true        | An access token with the `packages:delete` and `packages:read` scopes that is authorized in the organization where the package versions to be deleted are. |
```